config/query-api.yml

```diff
@@ -68,16 +68,17 @@ paths:
 
   /v1/webhook:
```

Comment:
It occurs to me that we'll need to put auth in before we go live with this.

Reply:
I'll add a ticket in zenhub